### PR TITLE
Add `displayTickets` interface

### PR DESCRIPTION
**Pull request by theVinesh**
Intercom introduced tickets space recently. This would setup a public method with which end users will be able to present tickets space.

 - Adds `displayTickets` interface
 - Bumps version

### CODE REVIEW

File: intercom_flutter_platform_interface/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # Changelog
+## 1.3.2
```
Comment (deepak786) [inline intercom_flutter_platform_interface/CHANGELOG.md:3]: Add line break after "Changelog"

**Review by deepak786 — CHANGES_REQUESTED** [review]

File: intercom_flutter_platform_interface/lib/intercom_flutter_platform_interface.dart [hunk 2]
```diff
@@ -278,4 +278,9 @@ abstract class IntercomFlutterPlatform extends PlatformInterface {
   Future<void> displayConversation(String conversationId) {
     throw UnimplementedError('displayConversation() has not been implemented.');
   }
+
+  /// To display an Activity with your Messages content.
+  Future<void> displayTickets() {
+    throw UnimplementedError('displayMessages() has not been implemented.');
```
Comment (deepak786) [inline intercom_flutter_platform_interface/lib/intercom_flutter_platform_interface.dart]: "displayTickets()" has not been implemented.

File: intercom_flutter_platform_interface/lib/intercom_flutter_platform_interface.dart [hunk 3]
```diff
@@ -278,4 +278,9 @@ abstract class IntercomFlutterPlatform extends PlatformInterface {
   Future<void> displayConversation(String conversationId) {
     throw UnimplementedError('displayConversation() has not been implemented.');
   }
+
+  /// To display an Activity with your Messages content.
```
Comment (deepak786) [inline intercom_flutter_platform_interface/lib/intercom_flutter_platform_interface.dart]: "Submit, view, and get updates on all of your tickets from one centralized place in the Messenger."

**Review by deepak786 — APPROVED** [review]

**Comment by deepak786** [comment]
@theVinesh You can create a new PR about the web implementation and method channel from this branch. I will merge all together.

**Comment by theVinesh** [comment]
@deepak786 [Web](https://github.com/v3rm0n/intercom_flutter/pull/353) and [Mobile](https://github.com/v3rm0n/intercom_flutter/pull/352) Impls are done.

LMK once this is merged and a new version of `flutter_platform_interface` is published.

**Pull request merged**